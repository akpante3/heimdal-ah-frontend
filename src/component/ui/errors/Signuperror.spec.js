/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SignupError from './signupError';

describe('The Signup Component Test Suite', () => {
  it('should match the snapshot', () => {
    const tree = shallow(<SignupError />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
