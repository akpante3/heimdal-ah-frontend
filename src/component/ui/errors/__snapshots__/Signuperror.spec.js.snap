// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Signup Component Test Suite should match the snapshot 1`] = `
<div
  className="col-md-12 text-danger text-center"
/>
`;
